static/js/index.js

```diff
@@ -57,7 +57,15 @@ function renderIndexDropdown(listIndices) {
     if (Cookies.get('IndexList')) {
         selectedSearchIndex = Cookies.get('IndexList');
     }else {
-        selectedSearchIndex = sortedListIndices[0].index;
-        $("#index-btn span").html(sortedListIndices[0].index);
+        if(sortedListIndices.length > 1){
```

Comment:
It should be `(sortedListIndices.length > 0)` to check at least one index is present.

Reply:
Fixed, please check